plugins/modules/fusion_hap.py

```diff
@@ -166,12 +166,17 @@ def create_hap(module, fusion):
     hap_api_instance = purefusion.HostAccessPoliciesApi(fusion)
     changed = True
     if not module.check_mode:
+        if not module.params["display_name"]:
+            display_name = module.params["name"]
+        else:
+            display_name = module.params["display_name"]
```

Comment:
```suggestion
        display_name = module.params["display_name"] or module.params["name"]
```
Personally, I like this as it is shorter and easier to read :)